core/wg/wg_work.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+
+def disable_server(path):
+    return "error" in subprocess.getoutput(f"wg-quick down {path}")
+
+def enable_server(path):
+    return "error" in subprocess.getoutput(f"wg-quick up {path}")
+
+
+def make_wg_server_base(ip, endpoint_port, private_key):
+    return f"""[Interface]
+Address = {ip}.1/24
+ListenPort = {endpoint_port}
+PrivateKey = {private_key}
+
+"""
+
+def peer_for_wg_server_config(peer_name, public_key, preshared_key, shared_ips):
```

Comment:
also, consider using serialized model from `core.db.model_serializer.ConnectionPeer`
```suggestion
from core.db.model_serializer import ConnectionPeer
...
def peer_for_wg_server_config(peer: ConnectionPeer):
```
use of serialized data guarantees integrity and correctness

Reply:
My bad, forgot about it